src/sentry/static/sentry/app/components/list.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import theme from 'app/utils/theme';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+export const List = styled('ul')`
+  list-style: none;
+  padding: 0;
+  margin-bottom: ${space(2)};
+
+  ol& {
+    counter-reset: numberedList;
+  }
+`;
+
+type ListItemProps = {
+  icon?: string | React.ReactNode;
+  children?: string | React.ReactNode;
+  className?: string;
+};
+
+const IconWrapper = styled('span')`
+  display: flex;
+  margin-right: ${space(1)};
+
+  /* Give the wrapper an explicit height so icons are line height with the
+   * (common) line height. */
+  height: 16px;
+  align-items: center;
+`;
+
+export const ListItem = styled(({icon, className, children}: ListItemProps) => (
+  <li className={className}>
+    {icon && (
+      <IconWrapper>
+        {typeof icon === 'string' ? <InlineSvg src={icon} /> : icon}
```

Comment:
Since this is a new component can we only accept Icon components? I think the long term plan is to not have InlineSvg icons at at all.

Reply:
Done and done!